apps/pie-microsite/src/content/pages/foundations/animation.md

```diff
@@ -3,6 +3,7 @@ eleventyNavigation:
     key: Animation
     parent: Foundations
 title: Animation
+layout: layouts/coming-soon-layout.njk
 ---
 
 This page is coming soon!
```

Comment:
Does this want removing?

Reply:
ah you mean the title? Yeah kept for now deliberately

Reply:
I meant that on some other pages (e.g. [what is PIE?](https://github.com/justeattakeaway/pie/pull/43/files#diff-7be73aa516762a13b8a6f3aa0c9779054c4a9fb242a0bc468783702c4d2aee21)) you've removed the "This page is coming soon!" text and replaced it with the coming soon layout, whereas on this page you have both.